<launch>

  <arg name="pbstream_filename" />
  <arg name="initpose_frame_name" default= "base_link" />
  <arg name="cartographer_tracking_frame_name" default= "base_link" />
  <arg name="get_tf_from_robot_description" default= "false" />

  <node name="set_initpose" pkg="cartographer_ros_pr2" type="initpose_node" output="screen"
        args="-configuration_directory $(find cartographer_ros_pr2)/configuration_files
              -configuration_basename velodyne_spinal_localization.lua
              -pbstream_filename $(arg pbstream_filename)" >

    <param name ="initpose_frame_name" value ="$(arg initpose_frame_name)"/>
    <param name ="cartographer_tracking_frame_name" value ="$(arg cartographer_tracking_frame_name)"/>
    <param name ="get_tf_from_robot_description" value = "$(arg get_tf_from_robot_description)" />
  </node>
</launch>
